model/schema/controlledVocabulary.yaml

```diff
@@ -80,3 +94,18 @@ classes:
       name:
         required: true
         identifier: true
+
+  VocabularyTermSet:
+    is_a: AuditedObject
+    description: >-
+      A subset of terms from a Vocabulary that are valid for particular
+      applications
+    slots:
+      - name
+      - set_vocabulary
+      - set_description
```

Comment:
I know its a long name but would it be better to use vocabulary_term_set_description, in the future we might want to have generic "sets" of things.

Reply:
sure - should we do the same for set_vocabulary -> vocabulary_term_set_vocabulary ?

Reply:
@oblodgett - done, I changed set_vocabulary too